examples/tensorflow/object_detection/tensorflow_models/README.md

```diff
@@ -0,0 +1,172 @@
+Step-by-Step
```

Comment:
Should we remove this readme, most of the context duplicate with single model readme.